### ISSUE TITLE
API change: antialiasing in preview tracer

### ISSUE THREAD

**Issue by joaander** [glotzerlab/fresnel#74]
## Description

Simplify `Tracer.preview` and the convenience API `fresnel.preview` antialiasing options. Replace `aa_level = 0  [int]` with `antialias = True [bool]`. The default should be `True` because it looks significantly better it should only be disabled when necessary for performance. Also, the path tracer always antialiases.

## Preview

`aa_level=0`
![sphere-0](https://user-images.githubusercontent.com/2197568/53647762-3bad5480-3c0c-11e9-8ae7-562c19fbdcec.png)

`aa_level=1`
![sphere-1](https://user-images.githubusercontent.com/2197568/53647790-4667e980-3c0c-11e9-8b63-d18a48025b38.png)

`aa_level=2`
![sphere-2](https://user-images.githubusercontent.com/2197568/53647825-57b0f600-3c0c-11e9-8abe-03f080131a06.png)

`aa_level=3`
![sphere-3](https://user-images.githubusercontent.com/2197568/53647833-5c75aa00-3c0c-11e9-847e-1d7d5bc612ed.png)

`aa_level=4`
![sphere-4](https://user-images.githubusercontent.com/2197568/53647851-63042180-3c0c-11e9-86b1-c28237504cd4.png)

Nominally, the relative cost of antialiasing is `(aa_level + 1)**2`. In practice the measured cost is:

1. 3.7 (CPU)  /  2.1 (GPU)
2. 14.1 (CPU) / 6.3 (GPU)
3. 55 (CPU) / 22 (GPU)
4. 222 (CPU) / 87 (GPU)

I do not know why there is such a discrepancy from the nominal cost, especially on the CPU. `aa_level=3` is very expensive.

## Developer

This issue is recommended for first time fresnel developers.

## Questions to consider

* What level should `antialiasing = True` set?
* Should the user be able to choose specific levels, or should we hide that API entirely?

**Comment by joaander** [comment]
Make `True` set `aa_level=2`

**Comment by joaander** [comment]
Regarding my earlier confusion, the number of samples per pixel is actually ``2**(2*aa_level)``. Thus, an increase of 1 increases the number of samples by a factor of 4, consistent with the performance numbers I posted.

Per @bdice 's recommendation, we decided to use `aa_level=3` for `True`. This is 64 samples per pixel. If we want to put in the effort, we could test other numbers of samples between 16 and 64 to determine a good trade-off between speed and image quality.
